src/exprestive.coffee

```diff
@@ -3,6 +3,7 @@ camelCase = require 'camel-case'
 express = require 'express'
 fs = require 'fs'
 path = require 'path'
+{pluralize, singularize } = require 'inflection'
```

Comment:
Agreed, but I just realized a stray space got in there before the `}`. Leaving a note here for myself to remove it